inference-engine/src/mkldnn_plugin/memory_descs/cpu_memory_desc.h

```diff
@@ -68,6 +71,8 @@ class MemoryDesc {
     // Get memory upper bound if possible. Can be undefined
     virtual size_t getMaxMemSize() const = 0;
 
+    virtual size_t getPaddedElementsCount() const = 0;
```

Comment:
Do we really sure that this method is meaningful on this level? Does this method make sense for any type of Descriptor?

Reply:
>>> Do we really sure that this method is meaningful on this level?
not sure about this, but we use this method into ```Reorder``` so looks like it's applied for all levels
https://github.com/mandrono/openvino/blob/feature/new_descriptor_hierarchy/inference-engine/src/mkldnn_plugin/nodes/mkldnn_reorder_node.cpp#L83

>>> Does this method make sense for any type of Descriptor?
it's make sense for blocked descs, as i can see for ```MKLDNNMemoryDesc``` it also make sense because ```padded_dims``` field placed in common part not in ```format_desc```